sig/protos/temporal/api/enums/v1/command_type.rbs

```diff
@@ -0,0 +1,64 @@
+module Temporal
```

Comment:
I wonder if these `rbs` files should sit right next to their `rb` files

Reply:
From all the examples I saw people are putting them separately. Here's an example of a fully typed OS project — https://github.com/HoneyryderChuck/httpx. At this point I'm not aware of specific reasons for this. I think it makes sense to follow in this case unless there's strong reason not to

Reply:
:+1: Makes sense